autoprotocol/protocol.py

```diff
@@ -1181,9 +1181,14 @@ def acoustic_transfer(
         for x in transfers:
             x["volume"] = round(x["volume"].to("nl"), max_decimal_places)
 
-        return self._append_and_return(
-            AcousticTransfer([{"transfer": transfers}], droplet_size)
-        )
+        if transfers:
+            return self._append_and_return(
+                AcousticTransfer([{"transfer": transfers}], droplet_size)
+            )
+        else:
+            raise RuntimeError(
+                "At least one transfer must have a nonzero transfer volume."
+            )
```

Comment:
I think we should move this block to line 1180. and instead rewrite the conditional as `if not transfer`, and then raise the error. 

Then, we don't have to wrap the return in the `if` statement because that looks ugly

Reply:
the changes are made